library/systemd/src/lib/yast2/service_widget.rb

```diff
@@ -33,24 +55,24 @@ module Yast2
   #     end
   #
   #     def write
-  #       service_configuration.write
+  #       @service.save
   #     end
   #   end
   class ServiceWidget
     include Yast::I18n
     include Yast::Logger
     include Yast::UIShortcuts
-    # creates new widget instance for given service configuration
-    # @param service_configuration [Yast2::ServiceConfiguration] configuration holder
-    def initialize(service_configuration)
+
+    # creates new widget instance for given service
+    # @param service_configuration [Yast2::SystemService,Yast2::CompoundService] service
+    def initialize(service)
       textdomain "base"
-      @service_configuration = service_configuration
+      @service = service
     end
 
     # gets widget term
     # @return <Yast::Term>
     def content
-      # TODO: disabling invalid action and autostart or kick it out?
       Frame(
         _("Service Configuration"),
         VBox(
```

Comment:
Why not to use `HSpacing` instead of `Label(" ")` ?

Reply:
Well, I basically copy ui from ServiceStatus. Maybe it is needed in ncurses? I do not know